### DISCUSSION TITLE
How to have an increased hover area around icons

### DISCUSSION THREAD

**Discussion by tech-meppem** [visgl/deck.gl#5577] [Q&A]
I'm using icon layers, and I want to make it more user-friendly to hover over icons and interact with them.
I have all the functionality done, however, due to the icons shapes, they have holes in them.
When the user hovers over the holes in the icon, they are no longer hovering over the icon.

I managed to do a psuedo-fix, by making transparent circle icons on another icon layer, and making hovering them equal to hovering over the icons, however, this has a problem.
The minimum transparency they can be is 13 (out of 255), because any alpha value below that isn't rendered by the Icon layer, and the user can not hover over them. This means you can still barely see the circles:
![image](https://user-images.githubusercontent.com/54634811/110809411-c3f37500-827c-11eb-9d10-7724c963f5f7.png)

Is there a way to specify an area around the icons which counts as hovering?

**Comment by Pessimistress** [comment]
[alphaCutoff](https://deck.gl/docs/api-reference/layers/icon-layer#alphacutoff)

**Reply by tech-meppem** [reply]
> [alphaCutoff](https://deck.gl/docs/api-reference/layers/icon-layer#alphacutoff)

Thanks, that works! I missed that in the docs.

Is there a way to do it for GeoJSON points as well?
As, alphaCutoff is for the IconLayer.

**Reply by sclebal** [reply]
How are you planning to use alphaCutoff in GeoJSON points layer? Since the alpha of every pixel of a point is the same, you shouldn't need it.

**Reply by tech-meppem** [reply]
> How are you planning to use alphaCutoff in GeoJSON points layer? Since the alpha of every pixel of a point is the same, you shouldn't need it.

To have increased hover/click area around points when trying to click on them.

**Reply by sclebal** [reply]
You may be interested in [pickingRadius](https://deck.gl/docs/api-reference/core/deck#pickingradius) or managing the click event out of deck and using [pickObject](https://deck.gl/docs/api-reference/core/deck#pickobject). Hope it helps!
